src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -552,6 +533,90 @@ class AemConfig(
     @Internal
     fun isUniqueProjectName() = project == project.rootProject || project.name == project.rootProject.name
 
+    /**
```

Comment:
deprecated, because configure closure should be used / using named parameters is just easier than remembering order